src/locales/en.json

```diff
@@ -26,6 +26,8 @@
     "season": "Season",
     "section": "Section",
     "sent": "Sent",
+    "storage_for_docking_equip": "Winter storage of accessories",
+    "storage_for_trailers": "Summer storage of trailers",
```

Comment:
`trailer_summer_storage` is already in these keys, they should probably be merged?

Reply:
The Swedish translation seems a bit different, which one do you suggest keeping?
```JSON
    "storage_for_trailers": "Sommarförvaring av trailer",
    "trailer_summer_storage": "Släp sommarförvaring",
```

Reply:
"Släp sommarförvaring" seems to me the more natural one of these two